tests/test_phase2.py

```
@@ -0,0 +1,292 @@
+        target_function="success_task",
+        args=json.dumps([1, 2]),
+        kwargs=json.dumps({}),
+        state_name=EnqueuedState.NAME
+    )
+    redis_storage.enqueue(job)
+    dequeued_job = redis_storage.dequeue(["default"], timeout_seconds=1)
+    assert dequeued_job.id == job.id
+
+    redis_storage.acknowledge(job.id)
+    # Acknowledged jobs are removed from processing list
+    assert redis_client.lrange(f"pytaskflow:queue:processing", 0, -1) == []
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)